AmplifyPlugins/DataStore/Sources/AWSDataStorePlugin/Storage/SQLite/StorageEngineAdapter+SQLite.swift

```diff
@@ -321,9 +326,22 @@ final class SQLiteStorageEngineAdapter: StorageEngineAdapter {
                          paginationInput: QueryPaginationInput? = nil,
                          eagerLoad: Bool = true,
                          completion: DataStoreCallback<[M]>) {
+        completion(query(modelType,
+                         modelSchema: modelSchema,
+                         predicate: predicate,
+                         sort: sort,
+                         paginationInput: paginationInput,
+                         eagerLoad: eagerLoad))
+    }
+
+    func query<M: Model>(_ modelType: M.Type,
```

Comment:
maybe we can make this private or update `StorageEngineAdapter` protocol?

Reply:
good point, making this private since it's not used anywhere else